### PR TITLE
Add instapush recipe

### PR DESCRIPTION
**Pull request by proinsias**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by proinsias** [comment]
I confirm I am willing to be listed in the maintainer section.

**Comment by proinsias** [comment]
@conda-forge/staged-recipes – I believe this is ready for review now. Not sure why AppVeyor builds are failing for these PRs.

File: recipes/instapush/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: df527396584f58c00886305cfa67fe0a89d57f9fddffe3bc87e6d3856f013a55
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/instapush/meta.yaml]: This can be 
```suggestion
  number: 0
  noarch: python
```

File: recipes/instapush/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - instapush
+
+about:
+  home: "https://github.com/adamwen829/instapush-py"
+  license: MIT
+  license_family: MIT
+  license_file: # LICENSE  # When available via adamwen829/instapush-py#4
```
Comment (xhochy) [inline recipes/instapush/meta.yaml]: Then please bundle the license as part of the recipe.

File: recipes/instapush/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - requests
```
Comment (marcelotrevisani) [inline recipes/instapush/meta.yaml]:
```suggestion
```

File: recipes/instapush/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - instapush
+
+about:
+  home: "https://github.com/adamwen829/instapush-py"
+  license: MIT
+  license_family: MIT
+  license_file: # LICENSE  # When available via adamwen829/instapush-py#4
+  summary: "a python wrapper for instapush"
+  doc_url:
```
Comment (marcelotrevisani) [inline recipes/instapush/meta.yaml]:
```suggestion
```

**Comment by proinsias** [comment]
@xhochy and @marcelotrevisani – I've updated as per your suggestions.

**Pull request merged**